pkg/cmd/drtprod/configs/drt_scale.yaml

```diff
@@ -126,3 +124,11 @@ targets:
         flags:
           warehouses: 100000
           db: cct_tpcc
+      - script: "pkg/cmd/drtprod/scripts/tpcc_run.sh"
```

Comment:
just to avoid confusion, can we rename the script to be generate_tpcc_run.sh or something?

Reply:
Done